README.md

```diff
@@ -259,6 +259,42 @@ To change the config.json for the desktop app, create a config file which will b
 
 In the paths above, `$NAME` is typically `Riot`, unless you use `--profile $PROFILE` in which case it becomes `Riot-$PROFILE`.
 
+Running from Docker
+===================
+
+The Docker image can be used to serve riot-web as a web server. The easiest way to use 
+it is to use the prebuilt image:
+```bash
+docker run -p 80:80 vectorim/riot-web
```

Comment:
Is `vectorim` an existing account on Docker Hub already? It seems like `matrixdotorg` does exist, but `vectorim` does not... I agree it likely makes sense for Riot to be in a separate `vectorim`-like account, but we should create it first before publishing these docs if it hasn't been done yet. (Also, would `newvector` or something be better...?)

It's also quite possible `vectorim` already does exist and I just don't understand Docker Hub at all. 😅

Reply:
I'm the best at including relevant details and omitted the plan here, sorry. The plan is to nag people internally for opinions first and then merge this